WordPress/gutenberg
# Custom script distclean to remove node_modules of all packages

## Description

This PR adds a custom script `distclean` that will remove the `node_modules` folder of all packages so that when `npm install` is run cached version fo the dependencies aren't utilized; new version are fetched based on the definition in the `package.json` file. 
## How has this been tested?
1. Run `npm install` so that the `node_modules` directories get created for the respective packages. 
2. Go to the `gutenberg` project and notice that the `node_modules` folders exists with the dependencies. 
3. Run the custom script `npm run distclean`.
4. Notice that the `node_modules` folders no longer exist. 




## Screenshots 
An example of how the `node_modules` folder is populated before `npm run distclean` is run.

<img width="748" alt="Screenshot 2021-01-13 at 3 49 43 PM" src="https://user-images.githubusercontent.com/1509205/104509195-50ecbb80-55b7-11eb-9b54-6562d3371eeb.png">

## Types of changes

The primary change is the addition of a custom script to `gutenberg/package.json`

## Checklist:
